### PR TITLE
Added "Hyrule Warriors Coop Aspect Ratio" gfx pack

### PR DESCRIPTION
**Pull request by MelonSpeedruns**

### CODE REVIEW

File: Resolutions/HyruleWarriors_Coop_AspectRatio/patches.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[ProjectZ_219v208]
+moduleMatches = 0xE159AE91
+
+0x1009609C = .float $ratio
+0x10096124 = .float $ratio
+0x10096948 = .float $ratio
+0x10096974 = .float $ratio
```
Comment (slashiee) [inline Resolutions/HyruleWarriors_Coop_AspectRatio/patches.txt:7]: Are you sure we need to change all these values? I feel like two of them would be used just for the co-op mode.

Comment (MelonSpeedruns) [reply Resolutions/HyruleWarriors_Coop_AspectRatio/patches.txt:7]: I didn't check, but I got those values from the resolution pack, and that uses all 4 of them.

Comment (slashiee) [reply Resolutions/HyruleWarriors_Coop_AspectRatio/patches.txt:7]: Did you resolve it because you checked to make sure all 4 changes were necessary?

**Comment by slashiee** [comment]
@MelonSpeedruns Are you sure that we need to modify all the aspect ratios for this pack?

**Comment by slashiee** [comment]
I've just noticed numerous issues with this pack, such as location and folder naming. I feel like it's best to have this be added as part of the version 5 pack instead, where we have more control with overriding stuff.

**Pull request closed without merging**